pkg/caret/R/updates.R

```diff
@@ -1,5 +1,6 @@
-update.train <- function(object, param = NULL, ...)
+update.train <- function(object, param = NULL, forceRefit = FALSE, ...)
 {
+  if(forceRefit) param <- object$bestTune
```

Comment:
I think this should be `if(forceRefit & is.null(param))` or even `if(is.null(param))`